firmware/controllers/algo/airmass/speed_density_airmass.cpp

```diff
@@ -41,3 +41,21 @@ AirmassResult SpeedDensityAirmass::getAirmass(int rpm) {
 		map,	// AFR/VE table Y axis
 	};
 }
+
+float SpeedDensityAirmass::getMap(int rpm) const {
+	float fallbackMap;
+	if (CONFIG(enableMapEstimationTableFallback)) {
+		// if the map estimation table is enabled, estimate map based on the TPS and RPM
+		fallbackMap = m_mapEstimationTable->getValue(rpm, 100 * Sensor::get(SensorType::Tps1).value_or(0));
```

Comment:
"100" magic constant, can we please give it a name or introduce a helpful macro?

Reply:
magic is no longer magic :heavy_check_mark: